homework-submission/week1/src/App.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Todos from './components/Todos';
+import AddTodo from './components/AddTodo';
+import data from './sources/data.json'
+
+class App extends Component {
+  state = {
+    data
+  }
+    handler = e => {
+      let todos = this.state.data;
+      todos[e.target.id -1].done = !todos[e.target.id -1].done;
+      this.setState({
+        todos
+      })
+    }
+
+    addTodo = e => {
+      e.preventDefault();
+      const {data} = this.state;
+      const newTodo = {
+        id: data.length + 1,
```

Comment:
It's better to generate random id whether by using a module to do this (there're many options, one of them are `uuid`) or to generate random number by `Math.random()`